docs/semgrep-cloud-platform/github-pr-comments.md

```diff
@@ -29,26 +32,21 @@ Object.entries(frontMatter).filter(
 
 <!--  The entire process of setting up the GH comment is more than just "enabling it", ie. turning it on. Users have to set up the rules. So I changed the verb. -->
 
-Semgrep can create **pull request (PR) comments** in your GitHub repository. These comments inform your developers of **findings**, such as security issues, in their PRs. Semgrep can also provide remediation tips or code fixes that your developers can click to **commit** into their code directly.
+<DeploymentJourney />
 
-:::info Prerequisites
-- Pull request (PR) comments can only be set up through Semgrep Cloud Platform (SCP). [<i class="fas fa-external-link fa-xs"></i> Create an account](/semgrep-code/getting-started/#signing-in-to-semgrep-cloud-platform) to set up PR comments.
-- You must connect your GitHub organization (org) to Semgrep.
-    - For **GitHub Cloud**: See [<i class="fa-regular fa-file-lines"></i> Adding an organization](/semgrep-cloud-platform/getting-started/#adding-an-organization).
-    - For **GitHub Enterprise Server**: [<i class="fa-regular fa-file-lines"></i> Integrate Semgrep with GitHub Enterprise](/semgrep-cloud-platform/scm/#integrating-semgrep-cloud-platform-with-github-enterprise-or-gitlab-self-managed) and if you have a VPN or firewall, [set up your allowlists](/semgrep-cloud-platform/scm/#receiving-pr-or-mr-comments-in-your-vpn-or-on-premise-scm).
-- You must add or onboard a Semgrep project (repository) to SCP and it must complete at least one full scan on your default branch successfully.
-:::
-## Conditions for PR comment creation
-
-PR comments appear for the following types of scans under these conditions:
-
-<CommentTriggers />
+Semgrep can create **pull request (PR) comments** in your GitHub repository. These comments provide a description of the issue detected by Semgrep and may offer possible solutions. These comments are a means for security teams, or any team responsible for creating standards to help their fellow developers write safe and standards-compliant code.
 
 Automated comments on GitHub pull requests are displayed as follows:
 
 ![Screenshot of a GitHub PR comment](/img/semgrep-pull-request.png#bordered)
 **Figure** An inline GitHub pull request comment.
 
+## Conditions for PR comment creation
+
+PR comments appear for the following types of scans under these conditions:
+
+<CommentTriggers />
+
 ## Confirm account connection and access
 
 ### Confirm your Semgrep account's connection to GitHub
```

Comment:
Can we remove this second header? It's so similar to the top that it was a bit confusing to see

Reply:
Agreed, I've turned it into steps so it's clear what are actual steps and what is optional.